### DISCUSSION TITLE
Unable to login to supabase

### DISCUSSION THREAD

**Discussion by jeromepui** [supabase/supabase#6950] [Questions] [answered]
Logging in with github gives me this error:
https://app.supabase.io/?error=server_error&error_description=failed+to+connect+to+%60host%3Dprod-2.c7y43uscdbtq.ap-southeast-1.rds.amazonaws.com+user%3Dauth_admin+database%3Dpostgres%60%3A+server+error+%28FATAL%3A+remaining+connection+slots+are+reserved+for+non-replication+superuser+and+rds_superuser+connections+%28SQLSTATE+53300%29%29

**Comment by dev-hiangel** [comment]
I have a same issue.

This is what I got an error from network tab.

error message: No authorization token was found
error code: 401
error id: bea7079d-a3f9-4fe2-bced-cd3536e58448

**Comment by GaryAustin1** [comment, marked as answer]
The SB team is working on it according to a post on Discord.  Dashboard only.

On status site now:
https://status.supabase.com/

**Reply by GaryAustin1** [reply]
Restored.
